### DISCUSSION TITLE
Daily vs. Weekly Irrigation

### DISCUSSION THREAD

**Discussion by Sweti934** [jeroenterheerdt/HAsmartirrigation#252] [General]
Hi all,

as far as I understood this component, it requires the lawn to be irrigated on a daily basis. 

Now I live in germany and here we had several garden guys tell us that we should irrigate only once a week and that deep, so the roots can be penetrated and not irrigate every day for e.g. 30 seconds and inly water the top of the soil.

What are your experiences? Could this irrigation be set to accumulate the whole week and then determine the irriagtion need on weekends, or a specific day? 

Thanks so much for this component and this community!

Best regards,

Sven

**Comment by jeroenterheerdt** [comment]
Hey Sven, thanks for checking in. This integration makes no assumptions here. It will do daily calculations but that doesn't mean you have to water daily. I don't, because for my type of grass it is advised to water every 2 days. However, very hot conditions change that, and I would think that would be the same in Germany. That's the beauty of this integration, you can decide for yourself when to irrigate. If you don't irrigate then you don't reset the bucket and it will just keep accumulating. Otherwise, irrigate and reset the bucket. Makes sense?

**Comment by Sweti934** [comment]
Hei @jeroenterheerdt  awesome thank you so much for your reply. I will try to read the FAQ again, to get a better understanding of the "bucket". I naively thought, that the daily time or so is a must, and the bucket is some kind of factor? So I have to irrigate daily. 

What I could not get from https://github.com/jeroenterheerdt/HAsmartirrigation/wiki/Example-behavior-in-a-week is - what is the bucket exactly and how can i "adjust" that, e.g. for once a week or every 4th day and so on? Sorry for stupid questions! Thank you so much for this awesome integration!

**Reply by jeroenterheerdt** [reply]
Please read the docs first. It's all there. Let me know if you have other questions afterwards.

**Comment by Sweti934** [comment]
Ah thanks! As long as I don´t reset the bucket, the time will go up / down according to the rain amount. 

One more thing: Can I also integrate a "Real" Rain sensor in mm? My weather station provides the actual rain data daily.

**Reply by jeroenterheerdt** [reply]
Yes, just set up a sensor group with that sensor. You need a cumulative sensor (ie, daily precipitation). Also, it's not just precip but also evaporation that is taken into account.

**Reply by Sweti934** [reply]
Ah lol I thought the precipitaion was the forecast lol. Anywhere I can change the sensor after the setup? Thanks so much!

**Reply by jeroenterheerdt** [reply]
from this I am assuming you are not running the beta of V2, because the beta have a beautiful UI for you to work with (panel in the sidebar). V1 is kind of clunky, you would have to venture into updating storage files... (files in hidden `.storage` directory). I'd recommend moving to V2 now.

**Reply by Sweti934** [reply]
Awesome! Did that, and with V2 it is working fine so far! One more "Stupid Question" - > If I now added the precipitation sensor from my weather station (And today it rains roughly 5mm) - how does the OWM Forecast play into this? Is there any sensor in the integration here, that takes a look into the forecasted precipitation? Thanks so much!

**Reply by jeroenterheerdt** [reply]
No, there is no sensor but it's used to cover for the situation where it is very sunny when you calculate but rain is expected early morning for example. If you don't take into account the forecast they you will be watering your lawn in the middle of the rain...

**Reply by Sweti934** [reply]
Awesome thanks! Checked it today (it is raining today) and as soon as I recalculate the zone, it still increases the time hugely. Shouldn´t the time decrease as it rains? I provided my rain sensor as input it states e.g. 2,0mm but the time goes up still... ?

**Reply by jeroenterheerdt** [reply]
there is much more that goes into how long to irrigate than just rain. Just make sure to have your updates regularly and calculate only once per day.

**Comment by Sweti934** [comment]
Ah lol i clicked on calculate many Times today lol. Thanks for the hint !

Jeroen ter Heerdt ***@***.***> schrieb am Mo., 18. Sept.
2023, 18:17:

**Reply by jeroenterheerdt** [reply]
Ideally you set it to automatic. The buttons are there for initial set up / troubleshooting.
